### DISCUSSION TITLE
There is currently no session available in Command

### DISCUSSION THREAD

**Discussion by unbreak** [symfony/symfony#49876] [Q&A] [answered]
### Symfony version(s) affected

5.4.21

### Description

When im trying render twig view in command i got Exception "There is currently no session available in Command".

```php
public function __construct(Environment $twig){
   $this->twig = $twig;
   parent::__construct();
}

protected function execute(InputInterface $input, OutputInterface $output): int
{
   $template = $this->twig->load("test.html.twig");

   $output->writeln($template->render()); //this line throw Exception

  return Command::SUCCESS;
}
```

Exception in terminal:

<img width="533" alt="img-2023-03-30-16-05-16" src="https://user-images.githubusercontent.com/1422449/228862052-6cfc262e-a578-44e6-8221-cee8596a87bb.png">

### How to reproduce

Every simple, and empty Command with pure Symfony installation throw this exception.

**Comment by ghost** [comment, marked as answer]
There is no session in CLI script. The concept of session is meaningless in the cli context.
Your twig template probably uses `{{ app.request }}` object. If you want to use this template in the cli script you have to remove it.

**Comment by unbreak** [comment]
@javaDeveloperKid thank you. In TwigExtension i had one global using `RequestStack` to `getSession`.
